lib/scroll-view/test/index.test.js

```diff
@@ -6,7 +6,7 @@ import expect from 'expect';
 import sinon from 'sinon';
 
 describe('<ScrollView /> Test', () => {
-  jsdom();
+  jsdom({skipWindowCheck: true});
```

Comment:
it was added as true so that mocha jsdom doesn't complain if jsdom is called in multiple tests